src/app/component/CartItem/CartItem.container.js

```diff
@@ -148,13 +149,13 @@ export class CartItemContainer extends PureComponent {
             }
         } = this.props;
 
-        return variants.findIndex(({ sku }) => sku === itemSku);
+        return variants.findIndex(({ sku }) => sku === itemSku || itemSku.includes(sku));
```

Comment:
This check does not look right, child SKU might be literally any value.

Reply:
itemSku in configurable products is dynamically generated when user adds the product to the cart.
![Screenshot_20200826_143413_1](https://user-images.githubusercontent.com/18352350/91299038-b3305580-e7a9-11ea-9bbd-9253cb12eabc.png)
![Screenshot_20200826_143834](https://user-images.githubusercontent.com/18352350/91299213-f094e300-e7a9-11ea-95a2-ad11a7b8dc77.png)

Reply:
For example: I changed sku on test product blue and it changed in itemSku in user cart
![Screenshot_20200826_150212](https://user-images.githubusercontent.com/18352350/91301295-5898f880-e7ad-11ea-8527-c970d95f26ad.png)